mkdocs.yml

```diff
@@ -22,6 +22,7 @@ nav:
     - Getting Started:
       - Overview: extensions/get-started/overview.md
       - Your First Extension: extensions/get-started/your-first-extension.md
+      - Generator: extensions/get-started/generator.md
```

Comment:
Maybe this should be under `Extension Guides`?

Reply:
It seems that extension guides is for introduction of main/renderer extensions, like this PR https://github.com/lensapp/lens/pull/1427/files If generator is for developer to bootstrap new projects, maybe better in the get started section?

Reply:
Still not sure about that. @nevalla @stevejr @msa0311 wdyt?

Reply:
If generator produces working extension like VS Code generator does, then it's fine to have in get started section. But in our case we are cloning the repo containing all the bits and pieces so it's a bit redundant step there. But the generator might be mentioned in Next steps section, for example:

>  Extension Guides contain instruction to install extension generator tool and additional samples to get started with your own extension.

Reply:
Ok, fixed in https://github.com/lensapp/lens/pull/1411/commits/5d904eac56b8bed0dda416411d9bfb6a8cae944c